meson.build

```diff
@@ -960,7 +960,7 @@ conf.set10('HAVE_MICROHTTPD', have)
 want_libcryptsetup = get_option('libcryptsetup')
 if want_libcryptsetup != 'false' and not skip_deps
         libcryptsetup = dependency('libcryptsetup',
-                                   version : '>= 1.6.0',
+                                   version : '>= 2.0.1',
                                    required : want_libcryptsetup == 'true')
         have = libcryptsetup.found()
         have_sector = cc.has_member(
```

Comment:
Yeah, I agree with @medhefgo that it'd nice to simplify this.

... e.g. this check can prolly go away.

Reply:
> ... e.g. this check can prolly go away.

Not exactly sure what "this check" refers to, but I guess you meant the HAVE_LIBCRYPTSETUP_SECTOR_SIZE check.